demo/README.md

```diff
@@ -0,0 +1,40 @@
+# DataYoga Demo
+
+## Overview
+
+The demo compose file starts the Datayoga `http.receiver` job configured to write incoming records into PostgreSQL database. 
```

Comment:
add link to PostgreSQL (https://www.postgresql.org/)

Reply:
✅